src/Compilers/CSharp/Test/Semantic/Semantics/ConstantTests.cs

```diff
@@ -2798,35 +2903,6 @@ public static int Main()
             CreateCompilation(source).VerifyDiagnostics();
         }
 
-        // We used to return constant zero in unchecked conversion from a floating-point type to an integral type
-        // in a constant expression if the value being converted is not within the target type.
-        // The C# spec says that in this case the result of the conversion is an unspecified value of the destination type.
-        // Zero is a perfectly valid unspecified value, so that behavior was formally correct.
-        // But it did not agree with the behavior of the native C# compiler, that apparently returned a value that
-        // would resulted from a runtime conversion with normal CLR overflow behavior.
-        // To avoid breaking programs that might accidentally rely on that unspecified behavior
-        // we now match the native compiler behavior, and we are going to keep this behavior for compatibility.
-        [Fact, WorkItem(1020273, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/1020273")]
-        public void Bug1020273()
-        {
-            string source = @"
-using System;
- 
-class Program
-{
-    static void Main()
-    {
-        double aslocal = 65535.17567;
-        Console.WriteLine(""As local: {0}"", unchecked((short)aslocal));
```

Comment:
>(short)aslocal [](start = 55, length = 14)

This test passes.  At least it does on Intel.  AMD possibly not.  The result here is undefined and therefore this is not suitable for a unit test.